### ISSUE TITLE
[Build & Deploy] Deploy PRs (develop to master) seem to be broken

### ISSUE THREAD

**Issue by roadlittledawn** [newrelic/opensource-website#754]
## Description

When you open a PR from `develop` to `master` it does not deploy (at least several have not in the last couple weeks.

Following the [doc](https://github.com/newrelic/opensource-website/wiki/Build-And-Deploy#production-deployment), it seems that a production deploy should happen. Currently, we do not have access to opensource in Amplify to check the logs / see what's happening when these PRs are opened.

### Acceptance Criteria

- [x] Figure out why deployments from develop > master aren't working
- [x] provide a `fix`

**Comment by jpvajda** [comment]
#616

**Comment by nr-kkenney** [comment]
@roadlittledawn @jpvajda a couple questions / comments:

1.  should we split the project stats failing problem into its own ticket, since that may be resolve-able on its own? it looks like the failure is due to us enabling branch protections.
2. if we split this ticket, should we block the preview generation portion of this ticket on getting aws access from CSI?

**Comment by roadlittledawn** [comment]
> @roadlittledawn @jpvajda a couple questions / comments:
> 
> 1. should we split the project stats failing problem into its own ticket, since that may be resolve-able on its own? it looks like the failure is due to us enabling branch protections.
> 2. if we split this ticket, should we block the preview generation portion of this ticket on getting aws access from CSI?

yep agree on both points!

**Comment by nr-kkenney** [comment]
cool. in that case, ill let this be the preview generation ticket & ill create a new one for the failing workflow.

**Comment by nr-kkenney** [comment]
blocked by: https://newrelic.atlassian.net/browse/CSI-3205

**Comment by aswanson-nr** [comment]
I think the issue here refers to one caused by the branch protection in this repository, it has currently been mitigated by removing the protection rule. This can be fixed by changing the workflows to handle branch protection and reenabling the protections.

**Comment by jpvajda** [comment]
fixed when the team team removed branched protection rules that were causing this error.
